medicines/doc-index-updater/src/pars_upload.rs

```diff
@@ -14,26 +14,35 @@ use std::collections::HashMap;
 use uuid::Uuid;
 use warp::{
     filters::multipart::{FormData, Part},
+    http::{header, Method},
     Filter, Rejection, Reply,
 };
 
 pub fn handler(
     state_manager: StateManager,
 ) -> impl Filter<Extract = impl Reply, Error = Rejection> + Clone {
+    let cors = warp::cors()
+        .allow_any_origin() // TODO restrict to a specific domain once we know what it is
```

Comment:
can we keep the restriction to `localhost` until we have other environments to which to restrict it? could introduce the environment variable here

Reply:
can we use our cdn endpoint here (+localhost for dev)?   https://mhraparsdev.azureedge.net